packages/angular_devkit/schematics/tools/file-system-engine-host-base.ts

```diff
@@ -321,10 +339,27 @@ export abstract class FileSystemEngineHostBase implements
       return factory();
     }
 
+    if (this._scheduler.has(name)) {
+      return observableOf(<T>(options: T | undefined, context: SchematicContext) => {
+        if (options !== undefined && !json.isJsonObject(options)) {
+          throw new UnregisteredTaskException(name);
+        }
+        const job = this._scheduler.schedule(
+          name,
+          options ? { ...JSON.parse(JSON.stringify(options)) } : context,
+          {
+            logger: context.logger,
+          },
+        );
+
+        return job.promise.then(() => {});
```

Comment:
`job.output.pipe(ignoreElements());`?

Reply:
That doesn't subscribe. Promise does.

Reply:
We also return the new `Promise<void>` which is necessary.

Reply:
It shouldn’t subscribe.  It should return a promise or observable.